packages/react-component-library/src/components/Timeline/README.md

```diff
@@ -250,11 +250,11 @@ const CustomTimelineComponent = ({
   const {
     width,
     offset,
-    isBeforeStart,
-    isAfterEnd
+    startsBeforeStart,
```

Comment:
I think this README is pretty stale now - it should probably be simplified and point to the microsite.

I have added a note to the board.